compiler-rs/clients_schema_to_openapi/src/paths.rs

```diff
@@ -311,6 +313,25 @@ fn get_path_parameters(template: &str) -> Vec<&str> {
     result
 }
 
+fn split_summary_desc(desc: String) -> SplitDesc{
+    let mut parts = desc.split(['.','\n',':']);
+    let first_line = parts.next().unwrap_or_else(|| "");
+    
+    let new_desc = desc.replace(first_line,"");
+    let trim = new_desc.trim();
+    let remove_period = trim.strip_prefix('.').unwrap_or_else(|| trim);
+    let remove_column = remove_period.strip_prefix(':').unwrap_or_else(|| remove_period);
```

Comment:
Nit 1: you can simplify things by using the same pattern as in `split()` above: `trim.strip_prefix(['.', ':']`).

Nit 2: making the variable mutable avoids declaring lots of additional variables:
```rust
let mut new_desc = ... // second part of `split_once` if it succeeded
let new_desc = new_desc.trim();
let new_desc = new_desc.strip_prefix('.').unwrap_or(new_desc);
let new_desc = new_desc.strip_prefix(':').unwrap_or(new_desc);

```